config_parser_test.cc

```diff
@@ -9,3 +12,184 @@ TEST(NginxConfigParserTest, SimpleConfig) {
 
   EXPECT_TRUE(success);
 }
+
+TEST(NginxConfigParserTest, VhostsConfig) {
+  NginxConfigParser parser;
+  NginxConfig out_config;
+
+  bool success = parser.Parse("vhosts_config", &out_config);
+
+  EXPECT_TRUE(success);
+}
+
+TEST(NginxConfigParserTest, ListenDefaultConfig) {
+  NginxConfigParser parser;
+  NginxConfig out_config;
+
+  bool success = parser.Parse("listen_default_config", &out_config);
```

Comment:
[Similar to above](https://github.com/apfong/nginx-configparser/pull/1/files#r96323085)